apps/src/applab/appStorage.js

```diff
@@ -372,7 +372,7 @@ var handlePopulateKeyValue = function (onSuccess, onError) {
     return;
   }
 
-  if (this.status != 200) {
+  if (this.status !== 200) {
```

Comment:
Actually, in this particular file it looks like we already elsewhere assume status is a number.